### PR TITLE
ath79: add support for NEC Aterm WF1200CR

### PR DESCRIPTION
**Pull request by musashino205**
- fix label_mac for NEC Aterm WG1200CR
  - https://github.com/openwrt/openwrt/commit/50fdc0374b75584f19c75f712ca0472a7113a065
  - https://github.com/openwrt/openwrt/pull/2138#issuecomment-509661135
- add support for NEC Aterm WF1200CR

### CODE REVIEW

File: target/linux/ath79/generic/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -303,6 +303,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan" "3:wan"
 		;;
+	nec,wf1200cr)
```
Comment (adschm) [inline target/linux/ath79/generic/base-files/etc/board.d/02_network:306]: wf before wg (sorting ...)

**Comment by adschm** [comment]
I picked the first patch. The second looks generally good from a first look as well ...

File: target/linux/ath79/generic/base-files/etc/board.d/02_network (hunk 1 above)
Comment (musashino205) [reply target/linux/ath79/generic/base-files/etc/board.d/02_network:306]: Oops, It's my mistake. I'll fix it.

**Comment by musashino205** [comment]
UPDATE:

- fix sorting of ```nec,wf1200cr``` in ```ath79_setup_interfaces``` in 02_network

**Comment by adschm** [comment]
Looks good from a second view as well.
Let's merge it.

**Pull request closed without merging**